.eslintrc.json

```diff
@@ -1,17 +1,22 @@
 {
+  "parserOptions": {
+    "ecmaVersion": 6,
+    "sourceType": "script"
+  },
```

Comment:
a bit tricky on this.. conflicts between Airbnb's config, Okonoet's config, `src/`, and `test/`

Reply:
what do you mean by conflicts? What's the issue exactly?